src/mgmt_conn.c

```diff
@@ -1320,7 +1338,16 @@ handle_start_rebuild_req(uzfs_mgmt_conn_t *conn, zvol_io_hdr_t *hdrp,
 
 	mutex_enter(&zinfo->main_zv->rebuild_mtx);
 
-	/* Check zinfo status */
+	// This migt be single replica case
+	if (can_start_rebuild_return_success(zinfo, req->volname)) {
```

Comment:
how about changing to can_START_REBUILD_return_success (or) can_return_success_for_START_REBUILD?
not sure though

Reply:
updated the func name as `start_rebuild_opcode_on_healthy_replica` as comment suggestion